common/cards/advent-of-tcg/single-use/inverter.ts

```diff
@@ -0,0 +1,33 @@
+import {SlotComponent} from '../../../components'
+import query from '../../../components/query'
+import {beforeAttack} from '../../../types/priorities'
+import {singleUse} from '../../defaults'
+import {SingleUse} from '../../types'
+
+const Inverter: SingleUse = {
+	...singleUse,
+	id: 'inverter',
+	name: 'Inverter',
+	expansion: 'default',
```

Comment:
This should probably be in the debug folder or similar

Reply:
agreed